src/qudi/hardware/ni_x_series/ni_x_series_finite_sampling_input.py

```diff
@@ -73,6 +74,7 @@ class NIXSeriesFiniteSamplingInput(FiniteSamplingInputInterface):
         name='external_sample_clock_frequency', default=None, missing='nothing')
 
     _physical_sample_clock_output = ConfigOption(name='sample_clock_output', default=None)
+    _invert_trigger_polarity = ConfigOption(name='invert_trigger_polarity', default=False)
```

Comment:
By using the constructor, we directly can get the _trigger_polarity to be `ni.constants.Edge.RISING` or `ni.constants.Edge.FALLING`.
For an example of using a constructor for a ConfigOption and an Enum see [here](https://github.com/Ulm-IQO/qudi-iqo-modules/blob/6cd2ff89143e6d45f2c5b35c09c91d2d73ab9818/src/qudi/hardware/ni_x_series/ni_x_series_finite_sampling_io.py#L94C1-L96C59)

For the sake of backwards compatibility, I would advise going with the default value of `RISING`, but since this can lead to unwanted behavior, I would opt for a `missing='warn'`